src/github.com/getlantern/balancer/balancer_test.go

```diff
@@ -132,7 +132,7 @@ func TestAll(t *testing.T) {
 		Dial: func(network, addr string) (net.Conn, error) {
 			atomic.StoreInt32(&dialedBy, 4)
 			defer atomic.AddInt32(&d4attempts, 1)
-			if d4attempts < 1 {
+			if atomic.LoadInt32(&d4attempts) < 1 {
 				// Fail once
 				return nil, fmt.Errorf("Me no dialee")
```

Comment:
Not a big deal, but I think this error string is a bit confusing, this is an intentional error, so something like "Failing intentionally." would be more appropriate.